src/helpers/qownnotesmarkdownhighlighter.cpp

```diff
@@ -134,6 +134,7 @@ void QOwnNotesMarkdownHighlighter::highlightMarkdown(const QString& text) {
     if (codeHighlightingOn) {
         if (previousBlockState() == HighlighterState::CodeBlock ||
             previousBlockState() == HighlighterState::CodeBlockEnd ||
+            previousBlockState() >= 200 ||
```

Comment:
Don't you want to use the constant here?

Reply:
Ah yes, just a sec